src/switch_core_media.c

```diff
@@ -5108,7 +5148,13 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 
 			if (switch_channel_var_false(smh->session->channel, "telnyx_disable_rtcp") || !switch_channel_get_variable(smh->session->channel, "telnyx_disable_rtcp")) {
 				if (remote_rtcp_port) {
-					if (switch_channel_var_true(smh->session->channel, "force_rtcp_passthru")) {
+					force_rtcp_passthru_set = switch_media_handle_test_media_flag(smh, SCMF_FORCE_RTCP_PASSTHRU);
```

Comment:
There are multiple check for `force_rtcp_passthru`, please cover all of those scenario